src/Table/Table.spec.tsx

```diff
@@ -0,0 +1,68 @@
+// VENDOR
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { ThemeProvider } from '@xstyled/styled-components';
+import { shallow, mount } from 'enzyme';
+// ANCHOR
+import { RootTheme } from '../../src/theme';
+import { Table } from './Table.component';
+const { Head, Body, Cell, Footer, Row, HeaderCell } = Table;
+
+describe('Component: Table', () => {
+    it('should be defined.', () => {
```

Comment:
`'should match snapshot'`. You can hoist the expect/tobedefined to the root describe block like I did with the widget.